### PR TITLE
add nextjs-8-serverless sample app

### PR DESCRIPTION
**Pull request by praveenweb**
### Description

This boilerplate makes use of the `serverless` mode in Next.js 8 to deploy each page as a lambda function.

What component does this PR affect? 

- [ ] Server
- [ ] Console
- [ ] CLI
- [ ] Docs
- [x] Community Content
- [ ] Build System

Requires changes from other components? If yes, please mark the components:

- [ ] Server
- [ ] Console
- [ ] CLI
- [ ] Docs
- [ ] Community Content
- [ ] Build System

### Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Docs update
- [x] Community content

### Checklist:

- [x] I have read the **[contributing guide](https://github.com/hasura/graphql-engine/blob/master/CONTRIBUTING.md)** and my code conforms to the guidelines.
- [ ] This change requires a change in the documentation. 
- [ ] I have updated the documentation accordingly.
- [ ] I have added required tests.

### CODE REVIEW

File: community/sample-apps/nextjs-8-serverless/README.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+```
+module.exports = {
+  target: "serverless",
+};
+```
+
+That's it! Now build the serverless app by running the following command:
+
+    ```bash
```
Comment (karthikvt26) [inline community/sample-apps/nextjs-8-serverless/README.md]: Not read properly

File: community/sample-apps/nextjs-8-serverless/README.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+pages/index.js => .next/serverless/pages/index.js
+pages/about.js => .next/serverless/pages/about.js
+```
+
+# Deploy to now.sh
+
+Deploy it to the cloud with [now](https://zeit.co/now) ([download](https://zeit.co/download)):
+
+```bash
+$ npm install -g now
```
Comment (shahidhk) [inline community/sample-apps/nextjs-8-serverless/README.md]: Can we remove $ from the commands? Harder to copy paste.

File: community/sample-apps/nextjs-8-serverless/README.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+```
+module.exports = {
+  target: "serverless",
+};
+```
+
+That's it! Now build the serverless app by running the following command:
+
+```
+$ npm run build
```
Comment (shahidhk) [inline community/sample-apps/nextjs-8-serverless/README.md]:
```suggestion
npm run build
```

File: community/sample-apps/nextjs-8-serverless/README.md (hunk 2 above)
Comment (shahidhk) [inline community/sample-apps/nextjs-8-serverless/README.md]:
```suggestion
npm install -g now
```

File: community/sample-apps/nextjs-8-serverless/README.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+pages/about.js => .next/serverless/pages/about.js
+```
+
+# Deploy to now.sh
+
+Deploy it to the cloud with [now](https://zeit.co/now) ([download](https://zeit.co/download)):
+
+```bash
+$ npm install -g now
+$ now
```
Comment (shahidhk) [inline community/sample-apps/nextjs-8-serverless/README.md]:
```suggestion
now
```

Comment (praveenweb) [reply community/sample-apps/nextjs-8-serverless/README.md]: The idea was to distinguish it as command line vs a code snippet.

**Pull request merged**